downstream/modules/platform/proc-create-controller-form-view.adoc

```diff
@@ -0,0 +1,19 @@
+[id="proc_create-controller-form-view_{context}"]
+
+= Create-{ControllerName}-form-view
+
+== Using the Form View
+
+. Ensure *Form view* is selected. It should be selected by default.
+. Enter the name of the new controller.
+. Add any labels necessary.
+. Click *Advanced configuration*.
+. Enter *Hostname* of the instance. The hostname is optional. The default hostname will be generated based upon the deployment name you have selected.
+. Enter the *Admin account username*.
+. Enter the *Admin email address*.
+. Select the *Admin password secret* from the dropdown menu.
+. Select *Database configuration secret* from the drop down menu.
+. Select the *Old Database configuration secret* from the drop down menu.
+. Select *Secret key secret* from the drop down menu.
+. Select *Broadcast Websocket Secret* from the drop down menu.
+. Enter any *Service Account Annotations* necessary.
```

Comment:
I haven't seen the UI for this, but it seems a little vague. Are there specific parameters for this? Maybe it's understandable to the end user?

Reply:
@jself-sudoku here is the UI:

![image](https://user-images.githubusercontent.com/11698892/193655106-e0f60552-0ba0-4eb2-bfc1-5723f03ef76d.png)

*Background*
Annotations can be added to any k8s resource, it is a way to add custom metadata about that object for debugging purposes.  In this case, it is for adding annotations to the service account object that is used for the AutomationController deployment.

An example of an annotation here would be:

```
  spec:
    service_account_annotations: |
      eks.amazonaws.com/role-arn: arn:aws:iam::<ACCOUNT_ID>:role/<IAM_ROLE_NAME>
```

Here is a more generic example:

![image](https://user-images.githubusercontent.com/11698892/193656380-16be0c29-02cc-46dc-9e3e-b9e343cd257a.png)

Reply:
More context on annotations can be found here: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#:~:text=Labels%20can%20be%20used%20to,characters%20not%20permitted%20by%20labels.